### ISSUE TITLE
Deeplink support 

### ISSUE THREAD

**Issue by vadi2** [Delwing/mudlet-map-browser-script#2]
It would be super if it was possible to share a link to a certain room in a certain area 👍

**Comment by Delwing** [comment]
It is. If you click the room, its number has a link to itself. E.g https://delwing.github.io/arkadia-mapa/?loc=1

**Comment by vadi2** [comment]
Ahh ok, my bad. I was just changing the areas and the url want updating. Could this be extended to areas as well?

**Comment by Delwing** [comment]
Certainly. Not documented anywhere, but it is there: https://delwing.github.io/arkadia-mapa?area=52

**Comment by vadi2** [comment]
Sweet. How did you create the link?

**Comment by Delwing** [comment]
You just need to put query param with your areaId from Mudlet's map.

**Comment by vadi2** [comment]
I don't have access to that on a phone 😅
